### PR TITLE
Added new category

### PR DESCRIPTION
**Pull request by salman0ansari**
added a new category "Temporary Service Bots" and added a new bot in that category.

### CODE REVIEW

**Review by FayasNoushad — CHANGES_REQUESTED** [review]
Add `---` on down of README

**Comment by salman0ansari** [comment]
> Add `---` on down of README

done

**Review by FayasNoushad — CHANGES_REQUESTED** [review]
Add in Table of Contents

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ A telegram bot list collection
 - [PDF Bots](#pdf-bots)
 - [Font Bots](#font-bots)
 - [Voice Chat Video Streamer Bots](#voice-chat-video-streamer-bots)
+- [Temporary Service Bots](temporary-service-bots)
```
Comment (FayasNoushad) [inline README.md:46]: Add hyperlink as hashtag

Comment (salman0ansari) [reply README.md]: Is everything good now?

**Review by FayasNoushad — APPROVED** [review]

**Pull request merged**